### PR TITLE
docs(pocketbase): add the integration page for standard OIDC

### PR DESCRIPTION
**Pull request by Tchoupinax**
Hello,

It is the creation of the page for integration [PocketBase](https://pocketbase.io/) with Authelia.
PocketBase is a "one backend file", an easy way to create backend applications. They support OIDC integration, that means all application using PocketBase can quickly integrate an OIDC workflow.
The integration is generic but in my case I tested with [UpSnap](https://github.com/seriousm4x/UpSnap) and it works well.

image

Thank you for your work
Have a nice day!

## Summary by CodeRabbit

- **Documentation**
	- Updated guidance on integrating Grafana with Authelia using OpenID Connect 1.0 to include PocketBase integration details.

### CODE REVIEW

File: docs/content/en/integration/openid-connect/pocketbase/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* __Authelia Root URL:__ `https://auth.example.com/`
+* __Client ID:__ `upsnap`
+* __Client Secret:__ `insecure_secret`
+
+## Configuration
+
+### Authelia
+
+The following YAML configuration is an example __Authelia__
+[client configuration](../../../configuration/identity-providers/openid-connect/clients.md) for use with [Grafana]
```
[the remark this answers is not in the document]
Comment (Tchoupinax) [reply docs/content/en/integration/openid-connect/pocketbase/index.md:41]: You are right

File: docs/content/en/integration/openid-connect/pocketbase/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+   6. User API URL: https://auth.example.com/api/oidc/userinfo
+   7. You can leave "Support PKCE" checked
+6. Save changes
+
+## See Also
+
+* [PocketBase OAuth Documentation](https://pocketbase.io/docs/authentication/#oauth2-integration)
+
+[Authelia]: https://www.authelia.com
+[Grafana]: https://pocketbase.io
```
[the remark this answers is not in the document]
Comment (Tchoupinax) [reply docs/content/en/integration/openid-connect/pocketbase/index.md:88]: Changed

File: docs/content/en/integration/openid-connect/pocketbase/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+To configure [PocketBase] to utilize Authelia as an [OpenID Connect 1.0], please follow this options:
+
+1. Connect to PocketBase admin view
+2. On the left menu, go to "Settings" (3rd icon)
+3. In "Authentication" section, go to "Auth providers"
+4. Select the gear on "OpenID Connect (oidc)"
+5. Configure:
+   1. ClientID: `upsnap`
+   2. Client secret: `insecure_secret`
+   3. Display name: `Authelia` (or whatever you want)
```
[the remark this answers is not in the document]
Comment (Tchoupinax) [reply docs/content/en/integration/openid-connect/pocketbase/index.md:76]: Good catch

[the remark this answers is not in the document]
Comment (james-d-elliott) [reply docs/content/en/integration/openid-connect/pocketbase/index.md]: I believe the AI is wrong in this instance.

File: docs/content/en/integration/openid-connect/pocketbase/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Tchoupinax) [reply docs/content/en/integration/openid-connect/pocketbase/index.md]: It should be good now, could you retry please?

**Comment by Tchoupinax** [comment]
The automatic review is pretty impressive

**Comment by james-d-elliott** [comment]
> The automatic review is pretty impressive

It is, has been pretty awesome. Picks up common mistakes or missing elements really well. Just comes down to style after that for docs at least.

**Review by james-d-elliott — APPROVED** [review]

**Pull request merged**